typescript/common.ts

```diff
@@ -1,5 +1,5 @@
 // From https://github.com/microsoft/TypeScript/issues/1897
-export type JSONPrimitive = string | number | boolean | null | unknown;
+export type JSONPrimitive = string | number | boolean | null | undefined;
 export type JSONValue = JSONPrimitive | JSONObject | JSONArray;
-export type JSONObject = { [member: string]: JSONValue };
+export type JSONObject = { [member: string]: JSONValue | any };
```

Comment:
Hmm, why is this needed? It seems to defeat the purpose of the JSONObject typing

Reply:
Take a look at https://lastmileai.dev/workbooks/clqnz6ebq00ekpew37w20ctzg. I can't really figure out a way around this

Reply:
Ah, darn. I think we might need to leave this for a future iteration and potentially move to using Record in place of JSONObject where applicable. JSONObject was intended to be a valid representation of what the object can be based on supporting JSON serialize/deserialize requirements but seems like it's too restrictive without the any/undefined